meetup-wordpress-medellin.php

```diff
@@ -130,3 +110,57 @@ function mwpm_get_headers( $report ) {
 
     return $headers;
 }
+
+function get_modal_form() {
+    // Inicializar variables y obtener los datos del usuario actual
+    $username   = '';
+    $user_email = '';
+
+    // Validar que el usuario esté loggeado.
+    if( is_user_logged_in() ) {
+        // Obtener los datos del usuario actual.
+        $current_user = wp_get_current_user();
+
+        $username   = $current_user->user_login;
+        $user_email = $current_user->user_email;
+    }
+
+    // Crear campo del nombre de quien reporta
+    $name_input  = '<label for="mwpm-report-form__name">' . esc_html__( 'Your Name', 'mwpm' ) . ': </label>';
```

Comment:
Creo que es mejor si toda la cadena se puede traducir de una vez, así podrían poner los `:` o el nombre de usuario al inicio si el lenguaje lo requiere (`rtl`vs `ltr`).

```suggestion
    /* translators: %s es el nombre de usuario del usuario actual. */
    $name_label  = sprintf( __( 'Your Name: %s', 'mwpm' ), $username );
    $name_input  = '<label for="mwpm-report-form__name">' . esc_html( $name_label ) . '</label>';
```